### PR TITLE
added getwaterml method/fixed conflict with erdc dev branch

### PR DESCRIPTION
**Pull request by msouff**
It was easier to close the other pull request and do this one from my dev to your dev branch.

### CODE REVIEW

**Comment by msouff** [comment]
Just added documentation.

File: docs/index.rst [hunk 1]
```diff
@@ -41,3 +41,4 @@ Contents
     setup/historical_lsm_process.rst
     setup/forecast_framework.rst
     setup/web_application.rst
+    SPT REST API/SPT REST API.rst
```
Comment (snowman2) [inline docs/index.rst]: Can you rename SPT REST API to api or rest_api? It is better not not have spaces in the name. Thanks!

Comment (snowman2) [reply docs/index.rst]: It is looking great by the way!

Comment (msouff) [reply docs/index.rst]: Done

**Comment by snowman2** [comment]
This looks great. Thanks for your contribution!

**Pull request merged**